### PR TITLE
[CI] Implemented deploy workflow

### LINKED ISSUE #118: Deploy on GitHub Pages

### PR DESCRIPTION
**Pull request by rmanaem**
- Closes #118 

Changes proposed in this pull request:

- Implemented deploy workflow
- Made a handful of UI changes

## Checklist
_This section is for the PR reviewer_

- [x] PR has an interpretable title with a prefix (`[ENH]`, `[FIX]`, `[REF]`, `[TST]`, `[CI]`, `[MNT]`, `[INF]`, `[MODEL]`, `[DOC]`) _(see our [Contributing Guidelines](https://neurobagel.org/contributing/CONTRIBUTING#pull-request-guidelines) for more info)_
- [x] PR has a label for the release changelog or `skip-release` (to be applied by maintainers only)
- [x] PR links to GitHub issue with mention `Closes #XXXX`
- [x] Tests pass
- [x] Checks pass

For new features:
- [ ] Tests have been added

For bug fixes:
- [ ] There is at least one test that would fail under the original bug conditions.

### CODE REVIEW

File: .github/workflows/deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Adapted from sample vite workflow file: https://vitejs.dev/guide/static-deploy#github-pages
+name: deploy
+
+on:
+  release:
+    types: [published]
+  push:
+    branches:
+      - feat-118
```
Comment (alyssadai) [inline .github/workflows/deploy.yaml:9]: should we remove this condition?

**Review by alyssadai — APPROVED** [review]
Looks good @rmanaem !! I think there might just be a leftover deploy condition in your wf.

Also, remember to add a PR label! I see that we actually haven't released this repo yet - should we just `release` on this PR as well?

Otherwise, 🧑‍🍳

File: .github/workflows/deploy.yaml (hunk 1 above)
Comment (rmanaem) [reply .github/workflows/deploy.yaml:9]: Yea will do before merge

**Comment by rmanaem** [comment]
> should we just release on this PR as well?

Yea sounds good.

**Pull request merged**